### ISSUE TITLE
share via instagram broken in ios < 9 

### ISSUE THREAD

**Issue by Bnaya** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#429]
```
  if (!IsAtLeastiOSVersion(@"9.0")) {
    if ([self canShareViaInstagram]) {
      CDVPluginResult * pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"not available"];
      [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
      return;
    }
  }
```
it should be 
```
    if (![self canShareViaInstagram]) {
``` 
https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/blob/master/src/ios/SocialSharing.m#L472

**Comment by EddyVerbruggen** [comment]
What exactly doesn't work for you on which platform?

**Comment by Bnaya** [comment]
When you use shareViaInstagram on iOS < 9,
you will get the error callback with "not available" error even its available
i've found the bug in that line

**Comment by EddyVerbruggen** [comment]
I see, thanks for reporting that!

**Comment by EddyVerbruggen** [comment]
Same issue for `shareViaWhatsApp` btw

**Comment by EddyVerbruggen** [comment]
FIxed in 5.0.4.

**Comment by Bnaya** [comment]
Thanks for the plugin :)
